### DISCUSSION TITLE
How to use specific RSU?

### DISCUSSION THREAD

**Discussion by lucianaswf** [orbisgis/geoclimate#926] [Q&A]
Hi!

Is it possible to run Geoclimate using a specific RSU, instead of those automaticaly generated? In my case I would like to use the Census tracks in order to use social and economic data with the results provided by Geoclimate.

Thanks!

**Comment by j3r3m1** [comment]
Hi,
Same problem as using your own data: it is possible using groovy scripts because the whole GeoClimate process is decomposed into several functions. But the script has not been coded yet. We hope to have a first script for the data mid-February, then you might be able to adapt it to your need.

**Comment by lucianaswf** [comment]
Okay. Thank you for the information. I look forward to this release.
